registry-api/generate-typescript.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+CURRENT_DIR=$(pwd)
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+CODEGEN_JAR=~/.ivy2/cache/io.swagger/swagger-codegen-cli/jars/
```

Comment:
I had to build a custom version of swagger-codegen to fix some problems.  Are you sure that isn't necessary anymore?
https://github.com/TerriaJS/magda/blob/master/tools/swagger-codegen-cli.md